lib/lorekeeper/multi_logger.rb

```diff
@@ -16,17 +16,38 @@ def inspect
       "Lorekeeper multilogger, loggers: #{@loggers.map(&:inspect)}"
     end
 
-if RUBY_VERSION > "2.6"
-    def respond_to?(method, all_included = false)
-      @loggers.all? { |logger| logger.respond_to?(method, all_included) }
-    end
-else
-    def respond_to?(method)
-      @loggers.all? { |logger| logger.respond_to?(method) }
+    # Define all common logging methods within Multilogger to avoid NoMethodError
+    def debug(*args, &block); call_loggers(:debug, *args, &block); end
+
+    def debug_with_data(*args, &block); call_loggers(:debug, *args, &block); end
+
+    def info(*args, &block); call_loggers(:info, *args, &block); end
+
+    def info_with_data(*args, &block); call_loggers(:info, *args, &block); end
+
+    def warn(*args, &block); call_loggers(:warn, *args, &block); end
+
+    def warn_with_data(*args, &block); call_loggers(:warn, *args, &block); end
+
+    def error(*args, &block); call_loggers(:error, *args, &block); end
+
+    def error_with_data(*args, &block); call_loggers(:error, *args, &block); end
+
+    def fatal(*args, &block); call_loggers(:fatal, *args, &block); end
+
+    def fatal_with_data(*args, &block); call_loggers(:fatal, *args, &block); end
+
+    if RUBY_VERSION > '2.6'
```

Comment:
@ykitamura-mdsol at this stage, can't we asume we are at least in 2.6 ?

Reply:
yeah, I think it's safe to drop ruby 2.5 support

https://github.com/JordiPolo/lorekeeper/blob/d3bcff1ae3c6600cafc79bec34cd801d4ea1b3a9/lorekeeper.gemspec#L23